cmd/kogito/command/shared/install_services.go

```diff
@@ -142,15 +142,15 @@ func (s *servicesInstallation) InstallMgmtConsole(mgmtConsole *v1alpha1.KogitoMg
 	return s
 }
 
-func (s *servicesInstallation) InstallOperator(warnIfInstalled bool, operatorImage string, force bool) ServicesInstallation {
+func (s *servicesInstallation) InstallOperator(warnIfInstalled bool, operatorImage string, force bool, channel KogitoChannelType) ServicesInstallation {
 	if s.err == nil && !s.operatorInstalled {
-		s.operatorInstalled, s.err = InstallOperatorIfNotExists(s.namespace, operatorImage, s.client, warnIfInstalled, force)
+		s.operatorInstalled, s.err = InstallOperatorIfNotExists(s.namespace, operatorImage, s.client, warnIfInstalled, force, channel)
 	}
 	return s
 }
 
 func (s servicesInstallation) SilentlyInstallOperatorIfNotExists() ServicesInstallation {
-	return s.InstallOperator(false, "", false)
+	return s.InstallOperator(false, "", false, AlphaChannel)
```

Comment:
same here.

Reply:
Use "ch" as reference variable to refer KogitoChannelType